moved/src/state_actor.rs

```diff
@@ -427,12 +435,20 @@ impl<
             .peek()
             .and_then(|(_, v, _)| v.as_deposited())
             .map(|tx| self.l1_fee.for_deposit(tx.data.as_ref()));
+        let l2_fee = self.l2_fee.with_gas_fee_multiplier(U256::from(1));
```

Comment:
Seems like a constant should be used here instead of a hardcoded literal. I think we maybe have a multiplier constant defined elsewhere already?

Reply:
We did have constants specific for Ecotone fees or block EIP-1559 fees, but I believe it would be a bit confusing to reuse any of them here. I added a const to the `CreateL2GasFee` in a similar fashion and a helper method. Hope it makes things more explicit.

Reply:
Yeah, looks good. Thanks!